automation_tools/scripts/ga-migration/templates/services/run-tests.sh

```diff
@@ -0,0 +1,30 @@
+
+#!/usr/bin/env bash
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or modify
```

Comment:
Could we change this to the generic Invenio license header? e.g. https://github.com/inveniosoftware/automation-tools/pull/9/files#diff-707cee7a3f79b3ef314738bb5c26a4fb63dc739a84046a96c0eaeed37374a374

Reply:
Yes, I have it locally, pushing soon.

Reply:
Pushed with some other improvements:

```
- Adapt to latest `docker-services-cli` version
- Fix various issues with Jinja white-spaces
- Emulate join function to generate EXTRAS
```